authzed/api/v0/namespace.proto

```diff
@@ -48,7 +48,23 @@ message Relation {
 }
 
 message TypeInformation {
-  repeated RelationReference allowed_direct_relations = 1;
+  repeated AllowedRelation allowed_direct_relations = 1;
```

Comment:
Commenting for clarity. This needs to only be compatible with the serialized proto format, which we've tested that this change is.